### ISSUE TITLE
[Paper Note] The Magic Correlations: Understanding Knowledge Transfer from Pretraining to Supervised Fine-Tuning, Simin Fan+, arXiv'26, 2026.02

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#4554]
# URL
- https://arxiv.org/abs/2602.11217
# Authors
  - Simin Fan
  - Dimitris Paparas
  - Natasha Noy
  - Binbin Xiong
  - Noveen Sachdeva
  - Berivan Isik
# Abstract
  - Understanding how language model capabilities transfer from pretraining to supervised fine-tuning (SFT) is fundamental to efficient model development and data curation. In this work, we investigate four core questions: RQ1. To what extent do accuracy and confidence rankings established during pretraining persist after SFT? RQ2. Which benchmarks serve as robust cross-stage predictors and which are unreliable? RQ3. How do transfer dynamics shift with model scale? RQ4. How well does model confidence align with accuracy, as a measure of calibration quality? Does this alignment pattern transfer across training stages? We address these questions through a suite of correlation protocols applied to accuracy and confidence metrics across diverse data mixtures and model scales. Our experiments reveal that transfer reliability varies dramatically across capability categories, benchmarks, and scales -- with accuracy and confidence exhibiting distinct, sometimes opposing, scaling dynamics. These findings shed light on the complex interplay between pretraining decisions and downstream outcomes, providing actionable guidance for benchmark selection, data curation, and efficient model development.
# Translation (by gpt-4o-mini)
- 言語モデルの能力が事前学習から教師ありファインチューニング（SFT）へどのように移行するかを理解することは、効率的なモデル開発とデータキュレーションにとって重要です。本研究では、次の4つの核心的な質問を調査します。RQ1. 事前学習中に確立された精度と信頼度のランキングは、SFT後にどの程度持続しますか？RQ2. どのベンチマークが堅固なクロスステージ予測因子として機能し、どれが信頼できませんか？RQ3. モデルのスケールが変わると、移行ダイナミクスはどのように変化しますか？RQ4. モデルの信頼度は精度とどの程度一致し、キャリブレーションの質の指標として機能しますか？この一致パターンは、トレーニングの各ステージに跨って移行しますか？これらの質問に対処するために、多様なデータミクスとモデルスケールに跨る精度および信頼度の指標に適用された一連の相関プロトコルを通じて探求します。実験の結果、移行の信頼性は能力カテゴリ、ベンチマーク、およびスケールによって劇的に異なり、精度と信頼度は異なる、時には opposing なスケーリングダイナミクスを示すことが明らかになりました。これらの発見は、事前学習の決定とその後の結果との複雑な相互作用に光を当て、ベンチマークの選定、データキュレーション、および効率的なモデル開発に向けた実用的なガイダンスを提供します。
# Summary (by gpt-4o-mini)
- 事前学習から教師ありファインチューニング（SFT）への移行を理解することは、モデル開発に重要。本研究では、モデルの精度と信頼度の持続性、信頼できるベンチマーク、スケールによる移行ダイナミクス、精度と信頼度の一致について調査。実験により、移行の信頼性は能力やベンチマーク、スケールによって異なり、精度と信頼度は異なるスケーリングダイナミクスを示すことが明らかに。これにより、ベンチマーク選定やデータキュレーションに関する実用的なガイダンスが提供される。

**Comment by AkihikoWatanabe** [comment]
元ポスト:https://x.com/rosinality/status/2022197251574214678?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q

**Comment by AkihikoWatanabe** [comment]
事前学習とSFTの間におけるAccuracyとConfidence（=モデルの回答のトークン確率）の相関を分析。モデルのスケールが大きい方が、SFT後のdownstreamタスクでのAccuracyと強い相関を持ち、confidence（=モデルが回答したときのトークンの確率）はモデルのスケールが小さい方が強い相関を持つ。このことから、よりモデルのスケールが大きい方がSFTにおいてAccuracyを維持するためにconfidenceの再形成を行っていることが示唆される、という話らしい。
<img width="1649" height="934" alt="Image" src="https://github.com/user-attachments/assets/e928a1e4-1c7c-4fbc-8145-f495515ba828" />
